src/common/types/row/tuple_data_allocator.cpp

```diff
@@ -5,6 +5,8 @@
 #include "duckdb/common/types/row/tuple_data_states.hpp"
 #include "duckdb/storage/buffer_manager.hpp"
 
+#include <duckdb/storage/buffer/block_handle.hpp>
```

Comment:
Looks like someone else also updated CLion ;)

Reply:
Ha, thanks for catching this. I hadn't noticed CLion was sabotaging me